tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -103,10 +87,18 @@ module.exports = {
     },
     /**
      * Checks to assert the notification bell is not present as well as no notifications are present
+     *
+     * @return boolean
      */
-    assertNoNotifications: function () {
-      return this.waitForElementNotPresent('@notificationBell')
-        .assert.elementNotPresent(this.elements.notificationElement)
+    isNotificationAbsent: async function () {
```

Comment:
I think it's better to just get the count of notifications and do the presence/absence logic  in the context

Reply:
It is done on context!

Reply:
@HariBhandari07 should we have different function to count notifications on page-object?

Reply:
I agree with @HariBhandari07 , making an `is...` function here, is basically the same as making the assertion. Let's have a counter function or use `getNotifications()` Wouldn't that return an empty array when there are no notifications, the context can count the size of the array

Reply:
Notifications are shown in a notification drawer which is opened and closed through a notification bell icon. Whenever a user gets notification then only the notification bell icon is shown on the nav-bar but when there are no notification we do not have that bell icon and hence we do not have any notifications on the notification-drawer. I've adjusted the function to check for the notification bell element to be present on the webUI and hence verify whether user have got notifications or not!